web/src/lib/components/AuctionCard.svelte

```diff
@@ -26,10 +26,12 @@
 
     function view() {
         if (auction.started) {
-            var viewedAuctions = (localStorage.getItem('auctions-viewed') || "").split(",");
+            var viewedAuctions = (browser && localStorage.getItem('auctions-viewed') || "").split(",");
```

Comment:
just FYI because we switched to SSR we need to check for `browser` before accessing `localStorage` because `localStorage` does not exist on the server side during rendering